controllers.js

```diff
@@ -1,9 +1,19 @@
-const {fetchTopics} = require('./models')
+const {fetchTopics, fetchArticleById} = require('./models')
 
 exports.getTopics = (req, res, next)=> {
     fetchTopics()
     .then(topics=>{
         res.status(200).send({topics});
     })
     .catch(next);
+}
+
+exports.getArticleById = (req, res, next)=>{
+    const ArticleId = req.params.article_id
+    fetchArticleById(ArticleId)
+    .then(article=>{
+//        console.log(article[0])
+        res.status(200).send(article[0]);
```

Comment:
Might be a subjective opinion, but I think it's nicer to access the first index in your array in your model